tests/unit/mock/config/compliance/compliance/ruckus_fastiron/fastiron_basic_backup.txt

```diff
@@ -0,0 +1,190 @@
+Current configuration:
+!
+ver 08.0.95gT211
+!
+stack unit 1
```

Comment:
Does the nesting ever go beyond 1 level of depth? If so, can you add that to the tests?

Reply:
It does for some L3 features like MP-BGP and VXLAN EVPN. We don't have a test device with such capabilities but I can definitely mock that in a test. Something like:

`overlay-gateway gateway1
 type layer2-extension
 ip interface Loopback 1
 map vlan 2 vni 3
 site site1
   ip address 67.67.67.1
   extend vlan add 2`